src/components/Swap/Toolbar/index.tsx

```diff
@@ -94,10 +117,52 @@ export default memo(function Toolbar() {
     inputAmount,
     isWrap,
     trade,
-    outputUSDC,
     impact,
+    open,
+    onExpand,
+    outputUSDC,
   ])
 
+  const tradeSummaryRows: SummaryRowProps[] = useMemo(() => {
+    const currencySymbol = trade?.outputAmount?.currency.symbol ?? ''
+    const rows: SummaryRowProps[] = []
+    rows.push({
```

Comment:
seems cleaner to just code them in the array directly instead of doing `.push` 3 times 🤔